### ISSUE TITLE
Add more explicit criteria for package inclusion

### ISSUE THREAD

**Issue by hoodmane** [pyodide/pyodide#1927]
We don't have any explicit criteria for when we would include a package as part of Pyodide. It would be good to add a PR template for adding packages that explains our criteria for inclusion (for instance, should a package be used a certain amount?).

**Comment by ryanking13** [comment]
I agree that we cannot include every package requested forever.
However, I am also afraid that setting a criteria may discourage people from adding new packages.
If we decide to set a creteria, I think it will be better to provide some other place where people can share their packages.

**Comment by hoodmane** [comment]
> I am also afraid that setting criteria may discourage people from adding new packages.

Yeah...

> If we decide to set criteria, I think it will be better to provide some other place where people can share their packages.

Perhaps the ideal thing would be if we could build binary wheels for wasm32_emscripten and then load them with micropip... but yeah this gets back into the whole future of Pyodide packaging topic which is itself a complicated ongoing discussion.

**Comment by rth** [comment]
> if we could build binary wheels for wasm32_emscripten and then load them with micropip..

I agree that would be ideal https://github.com/pyodide/pyodide/issues/655
